docs/UserGuide.adoc

```diff
@@ -11,226 +12,341 @@ ifdef::env-github[]
+
+****
+* The search is case insensitive. e.g `hans` will match `Hans`
+* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+* Only the name is searched.
+* Only full words will be matched e.g. `Han` will not match `Hans`
+* Employees matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+****
+
+Examples:
+
+* `find John` +
+Returns `john` and `John Doe`.
+* `find Betsy Tim John` +
+Returns any employee having names `Betsy`, `Tim`, or `John`.
+
+==== Listing all employees : `list`
+
+Shows a list of all employees in PTMan. +
+*Format:* `list` +
+*Shorthand:* `l`
+
+==== Listing entered commands: `history`
+
+Lists all the commands that you have entered in reverse chronological order. +
+*Format:* `history` +
+*Shorthand:* `h`
+
+[NOTE]
+====
+Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
+====
+
+==== Exiting the program: `exit`
+
+Exits the app. +
+*Format:* `exit`
+
+=== Employer Features
+These commands are privileged, and can only be executed by the employer. These commands requires the employer's admin password.
+
+[CAUTION]
+Admin password must always be entered as the *last* parameter of a command.
+
+==== Adding an employee: `add`
+
+Adds an employee to PTMan. +
+*Format:* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS s/SALARY [t/TAG]... pw/ADMIN_PASSWORD` +
+*Shorthand:* `a n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS s/SALARY [t/TAG]... pw/ADMIN_PASSWORD`
 
 [TIP]
-A person can have any number of tags (including 0)
+An employee can have any number of tags (including 0)
 
 Examples:
 
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 s/0 pw/DEFAULT1`  +
+Adds an employee named `John Doe` with phone number `98765432`, address `John street, block 123, #01-01`, and salary earned so far $`0`.
+* `add n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/1234567 s/100  pw/DEFAULT1`  +
+Adds an employee named `Betsy Crowe` with email `betsycrowe@example.com`, address `Newgate Prison`, phone number `1234567`, and salary earned so far $`100`.
 
-=== Listing all persons : `list`
 
-Shows a list of all persons in the address book. +
-Format: `list`
+==== Editing an employee: `edit`
 
-=== Editing a person : `edit`
+Edits an existing employee in PTMan. +
+*Format:* `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [s/SALARY] [t/TAG]... pw/ADMIN_PASSWORD` +
+*Shorthand:* `e INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [s/SALARY] [t/TAG]... pw/ADMIN_PASSWORD`
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+[NOTE]
+An employer password can't be edited. [refer to `rp` (reset password)]
```

Comment:
I can see `rp` only resets employee password. What about the employer? Can't be edited is acceptable, but can't be reset is a major gap!

Reply:
we rethought through and came out with a solution that allows both employee and employer to reset their password. I will be reflected in user guide.